src/webui/www/private/shareratio.html

```diff
@@ -0,0 +1,168 @@
+<!DOCTYPE html>
+<html lang="${LANG}">
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+    <title>QBT_TR(Torrent Upload/Download Ratio Limiting)QBT_TR[CONTEXT=UpDownRatioDlg]</title>
+    <link rel="stylesheet" href="css/style.css" type="text/css" />
+    <script type="text/javascript" src="scripts/mootools-1.2-core-yc.js" charset="utf-8"></script>
+    <script type="text/javascript" src="scripts/mootools-1.2-more.js" charset="utf-8"></script>
+    <script type="text/javascript" src="scripts/misc.js"></script>
+    <script type="text/javascript">
+        var UseGlobalLimit = -2;
+        var NoLimit = -1;
+
+        var keyboardEvents = new Keyboard({
+            defaultEventType: 'keydown',
+            events: {
+                'enter': function (event) {
+                    $('save').click();
+                    event.preventDefault();
+                }
+            }
+        });
+        keyboardEvents.activate();
+
+        window.addEvent('domready', function() {
+            var hashesList = new URI().getData('hashes').split('|');
+            var origValues = new URI().getData('orig').split(',');
+
+            var values = {
+                ratioLimit: friendlyFloat(origValues[0], 2),
+                seedingTimeLimit: parseInt(origValues[1]),
+                maxRatio: friendlyFloat(origValues[2], 2),
+                maxSeedingTime: parseInt(origValues[3])
+            };
+
+            // select current values
+            if (hashesList.length === 1) {
```

Comment:
how about `if (hashesList.length !== 1) return;`? and you save some indention.

Reply:
I've replaced this code.